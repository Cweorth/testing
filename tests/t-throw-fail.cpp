#include <cut.h>
#include <stdexcept>

TEST(throwFail) {
    throw std::runtime_error("expected to fail");
}