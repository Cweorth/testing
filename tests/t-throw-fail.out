b'[  1] throwFail............................................................\x1b[1;31mFAIL\x1b[0m\n    exception St13runtime_error: expected to fail\n\n\nSummary:\n  tests:       1\n  succeeded:   0\n  skipped:     0\n  failed:      1\n'